### PR TITLE
GC Subway: Fix index page display issue on desktop

### PR DESCRIPTION
**Pull request by GormFrank**
Also fixed sub-pages working examples layouts.

### CODE REVIEW

File: components/gc-subway/gc-subway.js [hunk 1]
```diff
@@ -44,14 +44,17 @@ var $document = wb.doc,
 			$h1 = $( "h1", $elm );
 			h1 = $h1.get( 0 );
 
-			// Ensure the element have an ID
-			if ( !h1.id ) {
-				h1.id = wb.getId();
-			}
+			if( h1 ) {
```
Comment (duboisp) [inline components/gc-subway/gc-subway.js:47]: Improve this condition, the 2 sub-instruction are only "if" conditional. For example `if ( h1 && !skipLink )`. But wait, is the variable "skipLink" is required? Is the init are called twice?

**Review by duboisp — CHANGES_REQUESTED** [review]

File: components/gc-subway/gc-subway.js (hunk 1 above)
Comment (GormFrank) [reply components/gc-subway/gc-subway.js:47]: @duboisp Yes, init gets called multiple times and this was causing multiple links to be added to the Skip links.

File: components/gc-subway/gc-subway.js [hunk 2]
```diff
@@ -44,14 +43,13 @@ var $document = wb.doc,
 			$h1 = $( "h1", $elm );
 			h1 = $h1.get( 0 );
 
-			// Ensure the element have an ID
-			if ( !h1.id ) {
-				h1.id = wb.getId();
-			}
+			// Add Subway H1 to skip links only once and if it is a sub-page
+			if ( h1 && event.currentTarget === event.target ) {
```
Comment (duboisp) [inline components/gc-subway/gc-subway.js:47]: Move up at line 41 the currentTarget check

**Review by duboisp — CHANGES_REQUESTED** [review]
After change completed and upon testing. It's pre-approved.

**Review by duboisp — APPROVED** [review]

**Comment by duboisp** [comment]
I tested and work as expected. I did a second code review and it looks good.

**Pull request merged**